src/components/LandDetailPage/LandDetailPage.types.ts

```diff
@@ -3,19 +3,24 @@ import { CallHistoryMethodAction } from 'connected-react-router'
 import { openModal, OpenModalAction } from 'modules/modal/actions'
 import { Deployment } from 'modules/deployment/types'
 import { ProjectState } from 'modules/project/reducer'
+import { LandTile } from 'modules/land/types'
 
 export type Props = {
   parcelsAvailableToBuildEstates: Record<string, boolean>
   deploymentsByCoord: Record<string, Deployment>
+  landTiles: Record<string, LandTile>
   onNavigate: (path: string) => void
   onOpenModal: typeof openModal
   projects: ProjectState['data']
 }
 
 export type State = {
-  hovered: string | null
+  hovered: Deployment | null
```

Comment:
Maybe this could be `isHovered`?

Reply:
mm this is not a boolean tho, it's the currently hovered `Deployment`

Reply:
I actually added this comment seeing the implementation and I completely missed the actual type here. My 🅱️